### PR TITLE
Dcase task1 Baseline

### PR DESCRIPTION
**Pull request by siddhu001**

### CODE REVIEW

**Comment by siddhu001** [comment]
@sw005320 @ftshijt I made this PR to add a baseline and data preparation code for Dcase task1. I believe this PR is ready to be merged. Please let me know any necessary changes that need to be made.

File: egs2/dcase_task1/asr1/local/data.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+stage=1
+stop_stage=100000
+log "$0 $*"
+. utils/parse_options.sh
+
+. ./db.sh
+. ./path.sh
+. ./cmd.sh
+
+DCASE="/ocean/projects/cis210027p/siddhana/TAU-urban-acoustic-scenes-2022-mobile-development/"
```
Comment (ftshijt) [inline egs2/dcase_task1/asr1/local/data.sh:24]: Could you add this entry to db.sh?

File: egs2/dcase_task1/asr1/local/data.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+if [ $# -ne 0 ]; then
+    log "Error: No positional arguments are required."
+    exit 2
+fi
+
+if [ -z "${DCASE}" ]; then
+    log "Fill the value of 'FSC' of db.sh"
```
Comment (ftshijt) [inline egs2/dcase_task1/asr1/local/data.sh:32]: Is it really FSC?

File: egs2/dcase_task1/asr1/run.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+test_sets="test"
+
+asr_config=conf/tuning/train_asr_hubert_transformer_adam_specaug.yaml
+inference_config=conf/decode_asr.yaml
+
+./asr.sh \
+    --lang en \
+    --ngpu 1 \
+    --stage 12\
+    --stop_stage 13\
```
Comment (ftshijt) [inline egs2/dcase_task1/asr1/run.sh:19]: Beter to remove

**Review by ftshijt — COMMENTED** [review]
Please also modify the entry in egs2/README.md

Btw, does Dcase would have different challenges data every year? If so, you may follow the iwslt convention by adding a year number to the recipe

File: egs2/dcase_task1/asr1/local/data.sh (hunk 2 above)
Comment (siddhu001) [reply egs2/dcase_task1/asr1/local/data.sh]: No this was typo, it should be DCASE. Good catch!

**Comment by siddhu001** [comment]
Thanks a lot @ftshijt for your feedback. Yes, I believe it makes sense to add year number to the recipe. I have addressed all your comments and incorporated your suggestions.

**Review by sw005320 — COMMENTED** [review]
LGTM.
My only request is to change the recipe name from `dcase_task1_2022` to `dcase22_task1`

**Comment by sw005320** [comment]
Thanks a lot, @siddhu001!

**Pull request merged**